### DISCUSSION TITLE
Ability to reference accounts by name?

### DISCUSSION THREAD

**Discussion by oldhatgaming** [dyc3/steamguard-cli#251] [Q&A] [answered]
I had about 25 accounts that I was using with SDA without issue. All of them migrated correctly to steamguard-cli, but whenever I use "steamguard code" I only get a code for the first account in my manifest.json. I'm assuming the codes would still be unique for the other 24 accounts that are still in my manifest.json.

Is there any way/plan to add code generation references by name from the list in manifest.json? Or does this already work and I don't understand how to generate codes for my other accounts?

Thanks

**Comment by dyc3** [comment, marked as answer]
Use `--username` or `-u` to specify which account to use, or `--all` to use all of them.

`steamguard --help` allows you to get the help text, and `steamguard <subcommand> --help` lets you see options for specific subcommands.
